css/style.css

```diff
@@ -13,10 +14,27 @@
   margin: 0;
 }
 
+#row {
+  width: 20em;
+  margin:auto;
+  font-size: 3em;
+}
+
+.col {
```

Comment:
The fact that row is an id and col is a class hurts my soul a little bit. I'd make them both classes.